x-pack/plugins/lens/public/indexpattern_plugin/indexpattern.tsx

```diff
@@ -55,16 +56,25 @@ export const indexPatternDatasource: Datasource<
   IndexPatternPersistedState
 > = {
   async initialize(state?: IndexPatternPersistedState) {
+    const indexPatternObjects = await getIndexPatterns();
+    const indexPatterns: { [id: string]: IndexPattern } = {};
+
+    if (indexPatternObjects) {
+      indexPatternObjects.forEach((obj: any) => {
```

Comment:
See comment from other `any` type - it also shouldn't be necessary here as the type from `getIndexPatterns` will be inferred here